omnibus.rb

```diff
@@ -44,9 +44,14 @@
 fetcher_retries 3
 fetcher_read_timeout 120
 
-# We limit this to 10 workers to eliminate transient timing issues in the
-# way Ruby (and other components) compiles on some more esoteric *nixes.
-workers 10
+# Work around for solaris zones, provide a less-bad number of workers
+if solaris? && Ohai['cpu']['total'] == 0
+  workers 10
+end
```

Comment:
Okay, the point here is to revert everyone other than Solaris to the `workers == ncpus+1` (IIRC) behavior, while maintaining this patch *ONLY* for Solaris.  Still kinda thing its specific to our Solaris servers, but at least now I get it...